data.json

```diff
@@ -65,6 +65,18 @@
             "furl": "https://zserge.com/rss",
             "hnp": 3050
         },
+        {
+            "name": "Dragoș Străinu blog",
+            "url": "https://strdr4605.github.io",
+            "tags": [
+                "software",
+                "hackathons",
+                "learning",
+                "frontend",
+                "open-source",
```

Comment:
thanks for the contribution, could you remove trailing comma

Reply:
```suggestion
                "open-source"
```